bar.go

```diff
@@ -53,8 +61,10 @@ func (b *bar) WindowRemoved(win Window) {
 	for i, icon := range icons {
 		if icon.taskbarWindow != nil {
```

Comment:
This is getting complex again. Let's invert the logic, i.e.

`if icon.taskbarWindow == nil || win != icon.taskbarWindow { continue }`